lib/SimpleHttpClient/ClientConnection.cpp

```diff
@@ -188,7 +185,7 @@ bool ClientConnection::readClientConnection(StringBuffer& stringBuffer, bool& co
       return true;
     }
 #ifdef ARANGODB_ENABLE_MAINTAINER_MODE
-    _read += (uint64_t)lenRead;
+    _read += static_cast<TRI_read_t>(lenRead);
```

Comment:
`_read` is a `uint64_t`, so casting to something else is pointless
```suggestion
    _read += static_cast<uint64_t>(lenRead);
```